src/components/common/TopDrops/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import withStyles from 'react-jss';
+
+import dropLeftA from '../../../assets/images/drops/drop_top_left_a.png';
+import dropLeftB from '../../../assets/images/drops/drop_top_left_b.png';
+import dropRightA from '../../../assets/images/drops/drop_top_right_a.png';
+import dropRightB from '../../../assets/images/drops/drop_top_right_b.png';
+import styles from './styles';
+
+const TopDrops = props => {
+  const { classes } = props;
+  return (
+    <div className={classes.root}>
+      <div className={classes.content}>
+        Fig. 1 - Our fields of speciality
```

Comment:
I would rather like to have this text and the line segment decoupled from the component since they are unrelated. They should be in the home page component. This component should only have the styles for the animations.